controllers/vmware/vspherecluster_reconciler_test.go

```diff
@@ -77,13 +78,13 @@ var _ = Describe("Cluster Controller Tests", func() {
 	// Ensure that the mechanism for reconciling clusters when a control plane machine gets an IP works
 	Context("Test controlPlaneMachineToCluster", func() {
 		It("Returns nil if there is no IP address", func() {
-			request := reconciler.VSphereMachineToCluster(vsphereMachine)
+			request := reconciler.VSphereMachineToCluster(context.TODO(), vsphereMachine)
```

Comment:
Let's please use context.Background() in cases like this.

The idea behind TODO is that at some point those context's shoudl be replaced by another one. I don't think this is necessary for these tests (please also check other places)